.github/pull_request_template.md

```diff
@@ -15,4 +15,6 @@ Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)
 ## Checklist:
 
 - [ ] I have performed a self-review of my own code
-- [ ] I have added tests that prove my fix is effective or that my feature works
\ No newline at end of file
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added test coverage to the JIRA ticket 
+- [ ] Commit contains the JIRA ticket number
```

Comment:
I'd argue that maybe only the merge commit should contain the ticket number, not the single commits from the merge request.

Reply:
These changes are just a cherry-pick Shaun's commit 
 - https://gitlab.motdev.org.uk/hvtesting/hvt-atf-availability/merge_requests/9#note_131724 I am okay with his approach but if you would like to change it, I think it would be worth the discussion.

Reply:
@bdrozd-kainos  internally I think this is right -  but here on github we do not need - [ ] I have added test coverage to the JIRA ticket 
- [ ] Commit contains the JIRA ticket number. For any external contributors - so can drop that 

@sam-artuso we have some tooling internally that reads commit messages into slack so it helps in the merge commit and that was the intention here